docs/writing-tests/accessibility-testing.mdx

```diff
@@ -130,18 +130,6 @@ If you enabled the addon and you're manually upgrading to Storybook 8.5 or later
 
 <CodeSnippets path="storybook-addon-a11y-test-setup.md" />
 
-### Override accessibility violation levels
```

Comment:
@kasperpeulen — Heads up that you'll need to merge `next` because I merged #30035, which updates this file.

Also, if we're removing this, should we document `reviewOnFail` instead?